src/main/java/hudson/plugins/audit_trail/AuditTrailPlugin.java

```diff
@@ -141,6 +151,19 @@ public void onFinalized(AbstractBuild build) {
                         " completed in " + build.getDuration() + "ms" +
                         " completed: " + build.getResult();
                 logger.log(message);
+                
+                if(logger instanceof LogFileAuditLogger){
```

Comment:
I feel like there is some separation of concern issue here, shouldn't ElasticSearch have its own `AuditLogger` where all the ES specific logic is handled? What do you think?